Meta/lint-shell-scripts.sh

```diff
@@ -4,6 +4,11 @@ set -e pipefail
 script_path=$(cd -P -- "$(dirname -- "$0")" && pwd -P)
 cd "$script_path/.."
 
+if ! command -v shellcheck >/dev/null 2>&1 ; then
```

Comment:
Small nitpick:
`if ! command -v shellcheck &>/dev/null ; then`

Reply:
Fixed.

`Meta/lint-clang-format.sh` also uses the same command :

```
Meta/lint-clang-format.sh:9:if command -v clang-format-10 >/dev/null 2>&1 ; then
Meta/lint-clang-format.sh:11:elif command -v clang-format >/dev/null 2>&1 ; then
```